packages/office-ui-fabric-react/src/components/Layer/examples/Layer.Customized.Example.tsx

```diff
@@ -47,7 +47,7 @@ export class LayerCustomizedExample extends React.Component<{}, ILayerCustomized
                 isClickableOutsideFocusTrap: true,
                 forceFocusInsideTrap: false
               }}
-              onDismissed={this._onDismissPanel}
+              onDismiss={this._onDismissPanel}
```

Comment:
This wasn't a type error? 😕

Reply:
Oh I see, it's a prop with differing functionality.

Reply:
Yep! onDismissed is for after the dismiss animation has completed. Probably just a typo.